Cargo.toml

```diff
@@ -27,6 +27,7 @@ paste = { version = "0.1.18", optional = true }
 pyo3-macros = { path = "pyo3-macros", version = "=0.14.1", optional = true }
 unindent = { version = "0.1.4", optional = true }
 hashbrown = { version = ">= 0.9, < 0.12", optional = true }
+indexmap = { version = "1.6.2", optional = true }
```

Comment:
Why is it fixed to the exact patch version? Could allow maybe `>= 1.0, < 2` here, perhaps.

Reply:
I fixed it to the latest version that was compatible with PyO3's MSRV. I will try something similar to hashbrown to have more flexibility